src/connectors/pagination/Paginator.ts

```diff
@@ -47,11 +58,15 @@ class Paginator {
     return padding;
   }
 
-  isLastPage() {
+  public nbPagesDisplayed(padding: number, total: number) {
```

Comment:
This looks like it could be private too since it's only used within this class?

Reply:
```suggestion
  private nbPagesDisplayed(padding: number, total: number) {
```